include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -34,21 +34,11 @@ namespace unseq_backend
 //This optimization depends on Intel(R) oneAPI DPC++ Compiler implementation such as support of binary operators from std namespace.
 //We need to use defined(SYCL_IMPLEMENTATION_INTEL) macro as a guard.
 
-template <typename _Tp>
-inline constexpr bool __can_use_known_identity =
-#    if ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION
```

Comment:
Is it okay if we directly remove this macro check, or should it be deprecated first with a `#warning` or something similar? From the user's perspective, they should just see `reduce_by_segment` speedup with 64-bit types.

Reply:
It is ok to directly remove it.